### ISSUE TITLE
[BUG] When switching to another app on Android, game restarts

### ISSUE THREAD

**Issue by OrionFive** [nickwalton/AIDungeon#246]
## 🐛 Bug Report

**Describe the bug**
When playing on Android, once you switch to a different app and back, the game begins with "Pick a setting" again, instead of continuing the current story. 

**How to Reproduce**

1. Play a story in the Android app
2. Switch to a different app
3. Switch back to AI Dungeon
4. See that it starts over instead of continuing the story

**Expected behavior**
The game should continue with the current story, as if the app was never left. 

**Additional context**
- Latest Android app (local machine?)

**Comment by steven4547466** [comment]
I believe that's known, but don't worry it saves. On the app, open the menu and go to "My Stories"  find the story (it should be at the top if it's the latest one), click on the story you want to continue and then play.

**Comment by OrionFive** [comment]
Yeah, this is more an inconvenience than a bug, I suppose. But when you often have to switch between apps it can be annoying to have to reload the story every time.

**Comment by Sporking** [comment]
This happens on iPhone also. Very annoying. Especially in light of bug #242.

**Comment by Sporking** [comment]
Clicking the "refresh" button also causes this problem. Very annoying, especially when the game appears to be stuck on some sort of network request, and refreshing the browser is the only way to regain control.
